app/utils/location.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios"
+import { loadString, saveString } from "./storage"
+import { CountryCode } from "libphonenumber-js/mobile"
+
+export async function getCountryFromIpAddress(): Promise<CountryCode> {
+  let countryCode = "SV" as CountryCode
+
+  try {
+    const response = await axios.get("https://ipapi.co/json/", {
+      timeout: 5000,
+    })
+    const _countryCode = response?.data?.country_code
+    if (_countryCode) {
+      await saveString("country_code", countryCode)
```

Comment:
we currently rely on apollo to store value locally, ie:

https://github.com/GaloyMoney/galoy-mobile/blob/fe5bdbd4013e373a8b0a13dccb5d4985943eb106/app/graphql/cache.ts#L79C11-L79C22
https://github.com/GaloyMoney/galoy-mobile/blob/fe5bdbd4013e373a8b0a13dccb5d4985943eb106/app/graphql/client-only-query.ts#L104-L116

Reply:
https://github.com/GaloyMoney/galoy-mobile/blob/fe5bdbd4013e373a8b0a13dccb5d4985943eb106/app/utils/theme-sync.tsx#L9

Reply:
Adjusted to use graphql cache instead. Code snippet was real useful